clients/python-wrapper/docs/conf.py

```diff
@@ -50,7 +50,7 @@
 
 def skip(app, what, name, obj, would_skip, options):
     if name in ('__init__',):
-        return False
+        return True
     return would_skip
```

Comment:
we can just `return name in ('__init__'):` (no ',' too)

Reply:
Removing this altogether since it's not necessary